### ISSUE TITLE
Trying to view a paid order throws an error

### ISSUE THREAD

**Issue by artistro08** [OFFLINE-GmbH/oc-mall-plugin#152]
When I try to view a paid order on my site, I get the error:
`No query results for model [OFFLINE\Mall\Models\Currency]. /vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php line 414` 

The stack trace it brings back is this

```

90 | Illuminate\Database\Eloquent\Builder->firstOrFail() | ~/plugins/offline/mall/models/Currency.php | 150
-- | -- | -- | --
89 | OFFLINE\Mall\Models\Currency::resolve(…) | ~/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php | 1489
88 | Illuminate\Database\Eloquent\Model::__callStatic(…) | ~/plugins/offline/mall/controllers/paymentlogs/_total.htm | 7
87 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
86 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
85 | Backend\Classes\Controller->makePartial(…) | ~/modules/backend/widgets/Lists.php | 1189
84 | Backend\Widgets\Lists->evalPartialTypeValue(…) | ~/modules/backend/widgets/Lists.php | 1046
83 | Backend\Widgets\Lists->getColumnValue(…) | ~/modules/backend/widgets/lists/partials/_list_body_row.htm | 27
82 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
81 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
80 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list_body_rows.htm | 2
79 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
78 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
77 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list.htm | 8
76 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
75 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
74 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list-container.htm | 2
73 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
72 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
71 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/Lists.php | 238
70 | Backend\Widgets\Lists->render() | ~/modules/backend/behaviors/relationcontroller/partials/_view.htm | 1
69 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
68 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/backend/classes/ControllerBehavior.php | 143
67 | Backend\Classes\ControllerBehavior->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
66 | Backend\Classes\ControllerBehavior->makePartial(…) | ~/modules/backend/behaviors/RelationController.php | 487
65 | Backend\Behaviors\RelationController->relationMakePartial(…) | ~/modules/backend/behaviors/RelationController.php | 429
64 | Backend\Behaviors\RelationController->relationRender(…) | ~/modules/backend/behaviors/RelationController.php | 474
63 | Backend\Behaviors\RelationController->relationRenderView() |   |  
62 | call_user_func_array(…) | ~/vendor/october/rain/src/Extension/ExtendableTrait.php | 409
61 | October\Rain\Extension\Extendable->extendableCall(…) | ~/vendor/october/rain/src/Extension/Extendable.php | 46
60 | October\Rain\Extension\Extendable->__call(…) | ~/modules/backend/behaviors/relationcontroller/partials/_container.htm | 15
59 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
58 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/backend/classes/ControllerBehavior.php | 143
57 | Backend\Classes\ControllerBehavior->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
56 | Backend\Classes\ControllerBehavior->makePartial(…) | ~/modules/backend/behaviors/RelationController.php | 487
55 | Backend\Behaviors\RelationController->relationMakePartial(…) | ~/modules/backend/behaviors/RelationController.php | 432
54 | Backend\Behaviors\RelationController->relationRender(…) |   |  
53 | call_user_func_array(…) | ~/vendor/october/rain/src/Extension/ExtendableTrait.php | 409
52 | October\Rain\Extension\Extendable->extendableCall(…) | ~/vendor/october/rain/src/Extension/Extendable.php | 46
51 | October\Rain\Extension\Extendable->__call(…) | ~/plugins/offline/mall/controllers/orders/show.htm | 58
50 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
49 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 109
48 | Backend\Classes\Controller->makeView(…) | ~/modules/backend/classes/Controller.php | 368
47 | Backend\Classes\Controller->execPageAction(…) | ~/modules/backend/classes/Controller.php | 246
46 | Backend\Classes\Controller->run(…) | ~/modules/backend/classes/BackendController.php | 135
45 | Backend\Classes\BackendController->run(…) |   |  
44 | call_user_func_array(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Controller.php | 54
43 | Illuminate\Routing\Controller->callAction(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php | 45
42 | Illuminate\Routing\ControllerDispatcher->dispatch(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Route.php | 212
41 | Illuminate\Routing\Route->runController() | ~/vendor/laravel/framework/src/Illuminate/Routing/Route.php | 169
40 | Illuminate\Routing\Route->run() | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 658
39 | Illuminate\Routing\Router->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 30
38 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php | 41
37 | Illuminate\Routing\Middleware\SubstituteBindings->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
36 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
35 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php | 49
34 | Illuminate\View\Middleware\ShareErrorsFromSession->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
33 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
32 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php | 63
31 | Illuminate\Session\Middleware\StartSession->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
30 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
29 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php | 37
28 | Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
27 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
26 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php | 66
25 | Illuminate\Cookie\Middleware\EncryptCookies->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
24 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
23 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 102
22 | Illuminate\Pipeline\Pipeline->then(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 660
21 | Illuminate\Routing\Router->runRouteWithinStack(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 635
20 | Illuminate\Routing\Router->runRoute(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 601
19 | Illuminate\Routing\Router->dispatchToRoute(…) | ~/vendor/october/rain/src/Router/CoreRouter.php | 20
18 | October\Rain\Router\CoreRouter->dispatch(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 176
17 | Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 30
16 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/plugins/offline/responsiveimages/classes/ResponsiveImagesMiddleware.php | 30
15 | OFFLINE\ResponsiveImages\Classes\ResponsiveImagesMiddleware->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
14 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
13 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/plugins/offline/speedy/classes/middleware/CDNMiddleware.php | 40
12 | OFFLINE\Speedy\Classes\Middleware\CDNMiddleware->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
11 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
10 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/plugins/offline/speedy/classes/middleware/Http2Middleware.php | 38
9 | OFFLINE\Speedy\Classes\Middleware\Http2Middleware->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
8 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
7 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php | 46
6 | Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
5 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
4 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 102
3 | Illuminate\Pipeline\Pipeline->then(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 151
2 | Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 116
1 | Illuminate\Foundation\Http\Kernel->handle(…) | ~/index.php

```
I tested this on https://mall.offline.swiss and got a different exception: 
`Illegal string offset 'name' /home/offlineg/public_html/_domains/mall.offline.swiss/plugins/offline/mall/controllers/paymentlogs/_payment_method.htm line 1`

Stack trace for that is:
```

81 | Raven_Breadcrumbs_ErrorHandler->handleError(…) | ~/plugins/offline/mall/controllers/paymentlogs/_payment_method.htm | 1
-- | -- | -- | --
80 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
79 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
78 | Backend\Classes\Controller->makePartial(…) | ~/modules/backend/widgets/Lists.php | 1189
77 | Backend\Widgets\Lists->evalPartialTypeValue(…) | ~/modules/backend/widgets/Lists.php | 1046
76 | Backend\Widgets\Lists->getColumnValue(…) | ~/modules/backend/widgets/lists/partials/_list_body_row.htm | 27
75 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
74 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
73 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list_body_rows.htm | 2
72 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
71 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
70 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list.htm | 8
69 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
68 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
67 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/lists/partials/_list-container.htm | 2
66 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
65 | Backend\Classes\WidgetBase->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
64 | Backend\Classes\WidgetBase->makePartial(…) | ~/modules/backend/widgets/Lists.php | 238
63 | Backend\Widgets\Lists->render() | ~/modules/backend/behaviors/relationcontroller/partials/_view.htm | 1
62 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
61 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/backend/classes/ControllerBehavior.php | 143
60 | Backend\Classes\ControllerBehavior->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
59 | Backend\Classes\ControllerBehavior->makePartial(…) | ~/modules/backend/behaviors/RelationController.php | 487
58 | Backend\Behaviors\RelationController->relationMakePartial(…) | ~/modules/backend/behaviors/RelationController.php | 429
57 | Backend\Behaviors\RelationController->relationRender(…) | ~/modules/backend/behaviors/RelationController.php | 474
56 | Backend\Behaviors\RelationController->relationRenderView() |   |  
55 | call_user_func_array(…) | ~/vendor/october/rain/src/Extension/ExtendableTrait.php | 409
54 | October\Rain\Extension\Extendable->extendableCall(…) | ~/vendor/october/rain/src/Extension/Extendable.php | 46
53 | October\Rain\Extension\Extendable->__call(…) | ~/modules/backend/behaviors/relationcontroller/partials/_container.htm | 15
52 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
51 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/backend/classes/ControllerBehavior.php | 143
50 | Backend\Classes\ControllerBehavior->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 97
49 | Backend\Classes\ControllerBehavior->makePartial(…) | ~/modules/backend/behaviors/RelationController.php | 487
48 | Backend\Behaviors\RelationController->relationMakePartial(…) | ~/modules/backend/behaviors/RelationController.php | 432
47 | Backend\Behaviors\RelationController->relationRender(…) |   |  
46 | call_user_func_array(…) | ~/vendor/october/rain/src/Extension/ExtendableTrait.php | 409
45 | October\Rain\Extension\Extendable->extendableCall(…) | ~/vendor/october/rain/src/Extension/Extendable.php | 46
44 | October\Rain\Extension\Extendable->__call(…) | ~/plugins/offline/mall/controllers/orders/show.htm | 58
43 | include(…) | ~/modules/system/traits/ViewMaker.php | 247
42 | Backend\Classes\Controller->makeFileContents(…) | ~/modules/system/traits/ViewMaker.php | 109
41 | Backend\Classes\Controller->makeView(…) | ~/modules/backend/classes/Controller.php | 368
40 | Backend\Classes\Controller->execPageAction(…) | ~/modules/backend/classes/Controller.php | 246
39 | Backend\Classes\Controller->run(…) | ~/modules/backend/classes/BackendController.php | 135
38 | Backend\Classes\BackendController->run(…) |   |  
37 | call_user_func_array(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Controller.php | 54
36 | Illuminate\Routing\Controller->callAction(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php | 45
35 | Illuminate\Routing\ControllerDispatcher->dispatch(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Route.php | 212
34 | Illuminate\Routing\Route->runController() | ~/vendor/laravel/framework/src/Illuminate/Routing/Route.php | 169
33 | Illuminate\Routing\Route->run() | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 658
32 | Illuminate\Routing\Router->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 30
31 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php | 41
30 | Illuminate\Routing\Middleware\SubstituteBindings->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
29 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
28 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php | 49
27 | Illuminate\View\Middleware\ShareErrorsFromSession->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
26 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
25 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php | 63
24 | Illuminate\Session\Middleware\StartSession->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
23 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
22 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php | 37
21 | Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
20 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
19 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php | 66
18 | Illuminate\Cookie\Middleware\EncryptCookies->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
17 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
16 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 102
15 | Illuminate\Pipeline\Pipeline->then(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 660
14 | Illuminate\Routing\Router->runRouteWithinStack(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 635
13 | Illuminate\Routing\Router->runRoute(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Router.php | 601
12 | Illuminate\Routing\Router->dispatchToRoute(…) | ~/vendor/october/rain/src/Router/CoreRouter.php | 20
11 | October\Rain\Router\CoreRouter->dispatch(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 176
10 | Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 30
9 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/plugins/bedard/debugbar/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php | 51
8 | Barryvdh\Debugbar\Middleware\Debugbar->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
7 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
6 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php | 46
5 | Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 149
4 | Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php | 53
3 | Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(…) | ~/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php | 102
2 | Illuminate\Pipeline\Pipeline->then(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 151
1 | Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(…) | ~/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php | 116
0 | Illuminate\Foundation\Http\Kernel->handle(…) | ~/index.php | 43

```

To reproduce the steps, I used the following. 

Checkout on frontend (using test Stripe Data) -> Going into backend to view order -> errors (above)

**Comment by tobias-kuendig** [comment]
That's strange. `$record->payment_method` returns the json string on mall.offline.swiss but a proper array on my local machine altough it is marked as `jsonable`. I guess the currency problem is the same.

I'll look into this.

**Comment by tobias-kuendig** [comment]
The problem was truncated json data. This should be fixed in 1.1.11. You might have to drop the invalid payment log entries manually after the update.

Would you mind testing the update and report your results back? Thank you!

**Comment by artistro08** [comment]
I did test it and looks like its working. thank ya! I didn't have to drop the payment logs.
